examples/replication/variables.tf

```diff
@@ -13,24 +13,15 @@ variable "prefix" {
 variable "resource_tags" {
```

Comment:
you never removed the default value for `prefix` in this example - lets be consistent